paket.lock

```diff
@@ -1699,54 +1699,54 @@ NUGET
-      NETStandard.Library (>= 1.6.1) - restriction: || (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
-      System.ComponentModel.TypeConverter (>= 4.3) - restriction: || (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
-      System.Runtime.Serialization.Formatters (>= 4.3) - restriction: || (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
-      System.Runtime.Serialization.Primitives (>= 4.3) - restriction: || (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
-      System.Xml.XmlDocument (>= 4.3) - restriction: || (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
+      Microsoft.CSharp (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
+      NETStandard.Library (>= 1.6.1) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
+      System.ComponentModel.TypeConverter (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
+      System.Runtime.Serialization.Formatters (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
+      System.Runtime.Serialization.Primitives (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard1.3)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6) (&& (== netstandard2.0) (< netstandard1.3))
+      System.Xml.XmlDocument (>= 4.3) - restriction: || (&& (== net46) (== netcoreapp2.0)) (&& (== net46) (== netstandard2.0)) (&& (== net46) (< net20)) (&& (== netcoreapp2.0) (< netstandard2.0)) (== netstandard1.6)
     Octokit (0.29)
       NETStandard.Library (>= 1.6) - restriction: || (&& (== net46) (< net45)) (== netcoreapp2.0) (== netstandard1.6) (== netstandard2.0)
     Paket.Core (5.145.1)
```

Comment:
Here it seems to be missing?

Reply:
I think that was a old commit. I have done a push -f ;)